### ISSUE TITLE
Colorful spots on models after #1271

### ISSUE THREAD

**Issue by tomimaki** [scp-fs2open/fs2open.github.com#1292]
After #1271  I get weird green and pink(?) spots on models, on some is more visible, on some less. Visible in F3 lab and mission. I tested Intel Mesa on linux and Nvidia on windows.
Before:
http://i.imgur.com/3AysdsW.png
http://i.imgur.com/iRenaOP.png
http://i.imgur.com/9ylcnVN.png
After:
http://i.imgur.com/BMFfbi9.png
http://i.imgur.com/WN8tE7U.png
http://i.imgur.com/eQd1zKP.png

**Comment by chief1983** [comment]
Hmm, I had been seeing some light spotting on the ISD hull in some places in FotG, thought it was specific to us but maybe not after all.

**Comment by The-E** [comment]
These discolorations are also present in 3.7.4. 

Here's current master with post processing turned off:
![image](https://cloud.githubusercontent.com/assets/280879/23668575/d008b948-0361-11e7-9d55-5f8905dba962.png)

And this is what it looks like under 3.7.4;
![image](https://cloud.githubusercontent.com/assets/280879/23668682/31b82f48-0362-11e7-9450-7c999f880493.png)

Based on what they look like, I am almost certain that these are DXT compression artefacts. Do any of you have uncompressed source maps to experiment with?

**Comment by chief1983** [comment]
Yeah I had suspected something like that with mine, and have not done a comparison to see if 1271 made them more apparent on the ISD.  But his case is pretty extreme looking for DXT artifacts I thought.

**Comment by TRBlount** [comment]
They're definitely artifacts. iirc, there was some difference loading the specular data between 3.7.4, the addition of deferred lighting or PBR, and #1271. This is probably the cause of the difference in artifacts appearing.

**Comment by The-E** [comment]
@TRBlount If you look at 3.7.5 with no post processing and 3.7.4, you have a largely identical picture by now (apart from the changes brought about by the new lightng code); it's only when you enable post processing (and, with it, HDR) that these artifacts become really visible. 

What I mean to say is, I don't know whether this is something that needs (or, for that matter, _can_) be addressed in code or whether it is a pure asset issue.

**Comment by SamuelCho** [comment]
It could be that the Fresnel in the new lighting model hid the compression artifacts but now that Fresnel is removed for -shine maps, the artifacts are more visible again?

**Comment by tomimaki** [comment]
Ah well. Long time after I used 3.7.4 build. And I see these artifacts too there.
So issue can be closed?
